pkg/standalone-utils/contracts/BalancerFeeBurner.sol

```diff
@@ -104,17 +104,33 @@ contract BalancerFeeBurner is IBalancerFeeBurner, ReentrancyGuardTransient, Vaul
         for (uint256 i = 0; i < steps.length; i++) {
             SwapPathStep memory step = steps[i];
```

Comment:
Per the change above, can just use the struct BatchRouterTypes

import { SwapPathStep } from "@balancer-labs/v3-interfaces/contracts/vault/BatchRouterTypes.sol";